service/user.service.js

```diff
@@ -4,14 +4,22 @@ const tokenService = require("../service/token.service");
 const UserDTO = require("../dto/user.dto");
 const ApiError = require("../helper/api.error");
 
-const signUp = async (name, email, password) => {
+const signUp = async (first_name, last_name, phone_number, email, password) => {
     const candidate = await UserModel.findOne({ email });
 
     if (candidate) throw ApiError.BadRequest("User already exist");
 
     const hashPass = bcrypt.hashSync(password, 10);
```

Comment:
@ikadushkina don't use blocking function as it blocks the Event Loop, bcrypt has an async analogue for hashing, use it.